bookclub.html

```diff
@@ -0,0 +1,121 @@
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+</head>
+
+<script>
+   var Airtable = require('airtable');
+// Get a base ID for an instance of art gallery example
+var base = new Airtable({ apiKey: 'keyUT2OSJVp7QqXmf' }).base('appwRoMbzzrnOc5PF');
+
+base('books').select({
+        sort: [
+            {field: 'Order', direction: 'asc'}
+        ]
+    }).eachPage(function page(records, fetchNextPage) {
+        records.forEach(function(record) {
+            console.log('Retrieved ', record.get('Name'), record.get("Url"));
+            var item = `<li><a href='${record.get("Url")}'>${record.get("Name")}</a> - ${record.get("Status")}</li>`
+            $('#bookList').append(item);
+
+            fetchNextPage();
+        });
+    }, function done(error) {
+        console.log(error);
+    });
+
+</script>
+
+<body id="page-top" data-spy="scroll" data-target=".navbar-fixed-top">
+    <!-- Navigation -->
+    <nav class="navbar navbar-custom navbar-fixed-top" role="navigation">
+        <div class="container">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
+                    Menu <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand page-scroll" href="#page-top">
+                  Bluegrass Developers Guild
+                </a>
+            </div>
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse navbar-right navbar-main-collapse">
+                <ul class="nav navbar-nav">
+                    <!-- Hidden li included to remove active class from about link when scrolled up past about section -->
+                    <li class="hidden">
+                        <a href="#page-top"></a>
+                    </li>
+                    <li>
+                      <a class="page-scroll" href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="#books">Books</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+    <!-- About Section -->
+    <section id="books" class="container content-section text-center">
+        <div class="row">
+          <div class="col-lg-8 col-lg-offset-2">
+            <h2>Book Club</h2>
+            <p>
+                <h3>Meetups are to learn things by practice <br/> book club is to learn things by studying</h3></p>
+            <p>Our purpose is to find books the group will enjoy and read, but learn something. It can be tech focused but not necessarily focused on tech</p>
+            <p>Below is our list of read and unread books. If you would like to take part join our slack channel.</p>
```

Comment:
`It can be tech focused but not necessarily focused on tech` not terribly clear

`If you would like to take part join our slack channel.` I think a comma after "part." Should it have a specific channel like #bookclub, or should "channel" be elided?

Reply:
Fixed